community/contributing/releasing/index.md

```diff
@@ -45,56 +47,15 @@ release branch. For example, all Spinnaker 1.16 releases (1.16.0, 1.16.1, etc.)
 are built from the `release-1.16.x` release branch. To get your patch into 1.16,
 it must be cherry-picked onto that release branch.
 
-There are two ways to create a pull request for a cherry-pick:
+After creating a pull request that you would like to be backported to a release
```

Comment:
```suggestion
After you've created a pull request to backport the fix to a release
```

Reply:
Well, I'm not sure this makes more sense, since the contributor-made PR itself isn't backporting the fix (the Mergify-generated PR will later, if it is approved by the release manager).

Reply:
Better?

Reply:
Love it! 🙏